doc/source/cluster/running-applications/job-submission/index.md

```diff
@@ -29,10 +29,7 @@ If needed, the Ray Jobs API also provides APIs for [programmatic job submission]
 
 ## Running Jobs Interactively
 
-If you would like to run an application *interactively* and see the output in real time (for example, during development or debugging), you can:
-
-- (Recommended) Run your script directly on a cluster node (e.g. after SSHing into the node using [`ray attach`](ray-attach-doc)), or
-- use [Ray Client](ray-client-ref) to run a script from your local machine while maintaining a connection to the cluster.
```

Comment:
I think you can keep this but say "For Experts only"

Reply:
should we keep the ray-client page then?